### ISSUE TITLE
protoc-gen-swagger: Incorrect description generated for google.protobuf.FieldMask

### ISSUE THREAD

**Issue by krosen040** [grpc-ecosystem/grpc-gateway#1556]
## 🐛 Bug Report

When generating a Swagger definition, the `description` for fields of type `google.protobuf.FieldMask` is not taken from the field's comment, as it is for other types, but is always `The set of field mask paths.`.

## To Reproduce

Create a `proto` file with a service that takes a message containing a `google.protobuf.FieldMask`:

```
syntax = "proto3";

package test;

import "google/api/annotations.proto";
import "google/protobuf/empty.proto";
import "google/protobuf/field_mask.proto";

service Foo {
    rpc Bar(Request) returns (google.protobuf.Empty) {
        option (google.api.http) = {
            get: "/foo"
        };
    }
}

message Request {
    // This should be generated as a description.
    google.protobuf.FieldMask a_mask = 1;
}
```

Generate the Swagger definition using `protoc -I. --swagger_out=...`

## Expected behavior

The `description` for the FieldMask parameter should be `This should be generated as a description.`.

## Actual Behavior

The `description` for the FieldMask parameter is `The set of field mask paths.`.

## Your Environment

go version go1.14.4 darwin/amd64
github.com/grpc-ecosystem/grpc-gateway v1.14.6

**Comment by johanbrandhorst** [comment]
Hi, thanks for the bug report! I expect this is because of the automatic field mask behavior. You can turn it off with a generator flag, but there might also be a fix in here somewhere. What can I do to help you contribute a fix for this? I'd start looking in https://github.com/grpc-ecosystem/grpc-gateway/tree/master/protoc-gen-swagger/genswagger. You could also add a comment to https://github.com/grpc-ecosystem/grpc-gateway/blob/master/examples/internal/proto/examplepb/a_bit_of_everything.proto#L322 and regenerate to reproduce the issue.

**Comment by krosen040** [comment]
@johanbrandhorst I haven't looked too deeply into this, but at a first glance it looks like the problem is that `google.protobuf.FieldMask` is not considered a well-known-type in the protoc-gen-swagger/protoc-gen-openapiv2 package. See [here](https://github.com/grpc-ecosystem/grpc-gateway/blob/master/protoc-gen-swagger/genswagger/template.go#L27).

When adding `google.protobuf.FieldMask` here, the correct description is being generated in the Swagger definition.

**Comment by johanbrandhorst** [comment]
Interesting, it might be as simple as that. We have tests testing the other fieldmask functionality, so hopefully if you open a PR with that change we can see if it breaks anything?

**Comment by krosen040** [comment]
I created a PR for this: #1558

**Comment by johanbrandhorst** [comment]
This was fixed in #1558
